server/plugins/OscUGens.cpp

```diff
@@ -2031,7 +2025,14 @@ void VOsc_Ctor(VOsc *unit)
 
 	unit->m_phasein = ZIN0(2);
 	unit->m_phaseoffset = (int32)(unit->m_phasein * unit->m_radtoinc);
-	unit->m_phase = unit->m_phaseoffset;
+
+	if (INRATE(2) == calc_FullRate) {
+		SETCALC(VOsc_next_ika);
+		unit->m_phase = 0;
```

Comment:
just curious: why do you want zero here instead of `ZIN0(2)`  (unit->m_phaseoffset)?